### DISCUSSION TITLE
The misorientation, express as angle<axes>

### DISCUSSION THREAD

**Discussion by SingPr** [mtex-toolbox/mtex#1558] [Ask Anything]
Hi everyone,
   Now I want to calculate the misorientation between the grain(1) and the grain(2), and express as angle<axes>. My codes are:
  ```matlab
ori1 = grains(1).meanOrientation
ori2 = grains(2).meanOrientation
axis(ori1, ori2)
angle12 = angle(ori1, ori2,'antipodal') ./ degree
```
 The running results are:
```
ans = vector3d (show methods, plot)
         x           y           z
  0.997486   0.0708476 -0.00168158

ang12 =

   17.5383
```
So, the misorientation between the grain(1) and the grain(2) is 17.5383°[0.997 0.070 -0.001] or 17.5383°<1 0 0>. Is that right?

Thanks
Sing

**Comment by ralfHielscher** [comment]
The angle computation is correct. The misorientation axis you computed is with respect to specimen coordinates. 
If this is what you want, its fine. Otherwise have a look at 
https://mtex-toolbox.github.io/orientation.axis.html to see how to compute the misorientation axis with respect to crystal coordinates.

Ralf.
